### PR TITLE
feat: add reviewed at to items

### PR DESCRIPTION
**Pull request by nicosantangelo**

### CODE REVIEW

File: src/entities/schema.ts [hunk 1]
```diff
@@ -195,7 +195,7 @@ export class Item extends Entity {
     this.set("rawMetadata", Value.fromString(""));
     this.set("isApproved", Value.fromBoolean(false));
     this.set("thirdParty", Value.fromString(""));
-    this.set("searchThirdPartyId", Value.fromString(""));
+    this.set("reviewedAt", Value.fromBigInt(BigInt.zero()));
```
Comment (LautaroPetaccio) [inline src/entities/schema.ts:198]: I think that removing `searchThirdPartyId` will break our code. I would suggest we keep it.

File: src/handlers/thirdparty.ts [hunk 2]
```diff
@@ -185,3 +183,26 @@ export function handleItemAdded(event: ItemAdded): void {
   let metric = buildCountFromItem()
   metric.save()
 }
+
+export function handleItemReviewed(event: ItemReviewed): void {
+  let itemId = buildItemId(event.params._thirdPartyId, event.params._itemId)
```
Comment (LautaroPetaccio) [inline src/handlers/thirdparty.ts:193]: There's a check we're doing when adding a new item that might be useful:
```
  if (!isBlockchainIdValid(event.params._itemId)) {
    log.error('An item was added in the TPR "{}" with an incorrect id "{}"', [
      event.params._thirdPartyId,
      event.params._itemId
    ])
    return
  }
```

File: src/handlers/thirdparty.ts [hunk 3]
```diff
@@ -163,14 +162,18 @@ export function handleItemAdded(event: ItemAdded): void {
...
+  if (item == null) {
     item = new Item(itemId)
   }
 
   item.blockchainItemId = event.params._itemId
   item.rawMetadata = event.params._metadata
   item.thirdParty = event.params._thirdPartyId
+  item.createdAt = event.block.timestamp
+  item.updatedAt = event.block.timestamp
+  item.reviewedAt = event.block.timestamp
```
Comment (LautaroPetaccio) [inline src/handlers/thirdparty.ts:174]: Should we have the same time for the `reviewedAt` and the `updatedAt` even if they weren't? Doesn't this overcomplicate the solution? That is, someone using the graph must know what if those values are equal, then it has not been updated nor reviewed, but that's not obvious.

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

File: src/handlers/thirdparty.ts (hunk 2 above)
Comment (nicosantangelo) [reply src/handlers/thirdparty.ts:193]: Oh! nice

File: src/handlers/thirdparty.ts (hunk 3 above)
Comment (nicosantangelo) [reply src/handlers/thirdparty.ts:174]: I'm following the implementation [here](https://github.com/decentraland/collections-graph/blob/master/src/handlers/collection.ts#L77) and more importantly [here](https://github.com/decentraland/builder/blob/master/src/modules/collection/utils.ts#L85) as we're trying to mimic collections. I'll have to use these values to fill up the TP collection so the front end works

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**